code/game/objects/items.dm

```diff
@@ -339,7 +339,7 @@
 
 		if(istype(src, /obj/item/clothing/under) || istype(src, /obj/item/clothing/suit))
 			if(FAT in H.mutations)
-				testing("[M] TOO FAT TO WEAR [src]!")
+				//testing("[M] TOO FAT TO WEAR [src]!")

```

Comment:
Why is this even output as debug text anyways? At the moment, it is most definitely being logged to the runtime log.